.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,96 @@
+name: "🐛 Bug Report"
+description: "If something isn't working as expected 🤔."
+title: "[Bug]: "
+labels: ["needs-triage"]
+body:
+  - type: markdown
+    attributes:
+      value: "## Please note the following potential times when an issue might be in Terraform core:"
+  - type: markdown
+    attributes:
+      value: |
+        * [Configuration Language](https://www.terraform.io/docs/configuration/index.html) or resource ordering issues
+        * [State](https://www.terraform.io/docs/state/index.html) and [State Backend](https://www.terraform.io/docs/backends/index.html) issues
+        * [Provisioner](https://www.terraform.io/docs/provisioners/index.html) issues
+        * [Registry](https://registry.terraform.io/) issues
+        * Spans resources across multiple providers
+
+        If you are running into one of these scenarios, we recommend opening an issue in the [Terraform core repository](https://github.com/hashicorp/terraform/) instead.
+  - type: textarea
+    attributes:
+      label: Community Note
+      description: Please leave this note as is for the community
+      value: |
+        * Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
+        * Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
+        * If you are interested in working on this issue or have submitted a pull request, please leave a comment
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Terraform CLI and Terraform AWS Provider Version
+      description: |
```

Comment:
Could we add placeholder attributes to show an example of our preferred format? I think they would be useful for most of the fields.

Reply:
I initially put placeholder values in for most of the `textarea` fields, but there's currently a bug in Forms that means that in Safari, the newlines are removed, so the formatting was off. Because of that, I removed them for now, but I'm happy to go back and re-add them so they're there when the bug is fixed if we'd prefer.

Reply:
Added placeholders -- I verified they look fine on Firefox and figure it's better to have them and deal with the [GitHub bug](https://github.com/github/feedback/discussions/4290) until it gets fixed in Safari.